exercises/rust-1-b/README.md

```diff
@@ -14,8 +14,11 @@ pub fn disemvowel(s: &str) -> String
 
 Note: in Rust, regular expressions are implemented in an [external crate](https://crates.io/crates/regex), and are thus not part of the standard library. As such, they are not available for this exercise.
 
-## Troubleshooting
+## Debugging
```

Comment:
```suggestion
## Debugging
```

Reply:
For some reason GitHub isn't parsing this properly.

Reply:
Yeah, I think that is actually a bug in the GitHub markdown renderer.